TIL/topics/dart_basics/StringControl.md

```
@@ -0,0 +1,161 @@
+		void main() {
+			print('Hello' + 'Dart'); // 'HelloDart'
+		}
+		```
+	- StringBuffer
+		- write() 메서드로 결합한 결과를 내부 메모리(버퍼)에 담아 두고 toString으로 결과를 얻는다.
+			```dart
+			final buffer = StringBuffer('Hello');
+
+			buffer
+				..write('World')
+			void main() {
```

markdownlint-cli2 (0.17.2)

Hard tabs
Column: 1

(MD010, no-hard-tabs)